PythonRegression/tests/features/steps/stitching_steps.py

```diff
@@ -1,12 +1,14 @@
 from aloe import step
 from util.test_logic import api_test_logic
+from util.transaction_utils import transaction_logic
 from iota import ProposedTransaction, Address, Tag, ProposedBundle
 import logging
 
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
 tests = api_test_logic
+transactions = transaction_logic
```

Comment:
Won't it be better to replace line 3 as "from util.transaction_utils import transaction_logic as transactions" instead of having "transactions  = transaction_logic". Same with lines 2 and 10.

Reply:
A fair point, yes it should.